centos_ci.sh

```diff
@@ -305,7 +305,7 @@ function build_and_test() {
   cd $GOPATH/src/github.com/minishift/minishift
   make prerelease synopsis_docs link_check_docs
   # Run integration test with 'kvm' driver
-  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags ~coolstore -format pretty"
+  MINISHIFT_VM_DRIVER=kvm make integration GODOG_OPTS="-tags flags,basic -format pretty"
```

Comment:
I am also intrigued about this. So far only coolstore was excluded, everything else ran. Now it is just basic and flags? What's about cmd-version, proxy and provision_earlier_version? Can we not stick to ~coolstore for now? Or have we reached a limit where the current timeout limit is reached?